### PR TITLE
[MIG][13.0] sale_stock_secondary_unit: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT21677
@carlosdauden @Tardo Review...

### CODE REVIEW

File: sale_stock_secondary_unit/models/stock_rule.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        product_id,
+        product_qty,
+        product_uom,
+        location_id,
+        name,
+        origin,
+        company_id,
+        values,
+    ):
+        res = super(StockRule, self)._get_stock_move_values(
```
Comment (Tardo) [inline sale_stock_secondary_unit/models/stock_rule.py]:
```suggestion
        res = super()._get_stock_move_values(
```

File: sale_stock_secondary_unit/models/stock_rule.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            product_id,
+            product_qty,
+            product_uom,
+            location_id,
+            name,
+            origin,
+            company_id,
+            values,
+        )
+        if values.get("sale_line_id", False):
```
Comment (Tardo) [inline sale_stock_secondary_unit/models/stock_rule.py]: Default is 'None' and is evaluated as falsy value too
```suggestion
        if values.get("sale_line_id"):
```

**Review by Tardo — APPROVED** [review]
:+1: LGTM. non-blocking suggestions.

See travis dependencies problem.

**Comment by sergio-teruel** [comment]
@Tardo Changes done!!
This PR denpends on https://github.com/OCA/stock-logistics-warehouse/pull/796
Can you review it??

**Comment by Tardo** [comment]
All OK for me... but travis continue with error status :/

**Review by ThomasBinsfeld — APPROVED** [review]

**Pull request merged**